static/js/components/ImageUploaderForm.js

```diff
@@ -122,6 +57,80 @@ export default class ImageUploaderForm<Form> extends React.Component<
     formValidate({ image: message })
   }
 
+  uploaderBodyHeight = (): number => {
```

Comment:
This could just be a `const` now instead of a class method since it's not using `this.props` anymore.